redirects/16.0.txt

```diff
@@ -15,6 +15,9 @@ applications/finance/payment_acquirers/paypal.rst applications/finance/payment_p
 applications/finance/payment_acquirers/sips.rst applications/finance/payment_providers/sips.rst                                       # /payment_acquirers/* -> /payment_providers/*
 applications/finance/payment_acquirers/stripe.rst applications/finance/payment_providers/stripe.rst                                   # /payment_acquirers/* -> /payment_providers/*
 applications/finance/payment_acquirers/wire_transfer.rst applications/finance/payment_providers/wire_transfer.rst                     # /payment_acquirers/* -> /payment_providers/*
+applications/finance/accounting/others/analytic/usage.rst applications/finance/accounting/others/analytic/analytic_accounting.rst     # /analytic/usage --> /analytic/analytic_accounting
+applications/finance/accounting/others/analytic/timesheets.rst applications/finance/accounting/others/analytic/analytic_accounting.rst # /analytic/timesheets --> /analytic/analytic_accounting
+applications/finance/accounting/others/analytic/purchase_expenses.rst applications/finance/accounting/others/analytic/analytic_accounting.rst # /analytic/purchase_expenses --> /analytic/analytic_accounting
```

Comment:
Move these before `applications/finance/payment_acquirers...` in alphabetical order so that it's easier to track and manage during forward-ports.